projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-button.php

```diff
@@ -7,12 +7,12 @@
  * @package automattic/jetpack
  */
 
-namespace Automattic\Jetpack\Extensions\Sharing_Button;
+namespace Automattic\Jetpack\Extensions\Sharing_Button_Block;
 
 use Automattic\Jetpack\Blocks;
 use Jetpack_Gutenberg;
 
-require_once JETPACK__PLUGIN_DIR . 'modules/sharedaddy/sharing-sources.php';
+require_once JETPACK__PLUGIN_DIR . 'modules/sharedaddy/sharing-sources-block.php';
```

Comment:
I copied original `sharing-sources.php` and left only the logic we possible need (URLs and Redirects). Not sure where we should keep it though. As a new Module, as a Package (with only 1 file) or I can simply put this php file in the block folder and use it as sort of utils?

Reply:
>  I can simply put this php file in the block folder and use it as sort of utils?

That would be my vote, I like that idea. 👍 I think it would also make it easier for new folks to understand and onboard on the long term.